test/ergw_ggsn_test_lib.erl

```diff
@@ -431,14 +431,17 @@ validate_seq_no(#gtp{seq_no = SeqNo}, #gtpc{seq_no = GtpCSeqNo}) ->
 validate_seq_no(#gtp{seq_no = SeqNo}, ExpectedSeqNo) ->
     ?equal(ExpectedSeqNo, SeqNo).
 
+validate_teid(#gtp{tei = 0} , #gtpc{}) ->
+    ok;
```

Comment:
don't do that. Testing for zero TEID should remain explicit in the places where such a TEID is indeed expected.